mod_reforged/hooks/items/tools/faction_banner.nut

```diff
@@ -0,0 +1,8 @@
+::mods_hookExactClass("items/tools/faction_banner", function(o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+		this.m.Reach = 5;
```

Comment:
Setting the Reach to 6 and removing the custom exception in Bolster seems a much better solution